READMEs/c2d-flow.md

```diff
@@ -0,0 +1,398 @@
+
+assert alice_wallet.web3.eth.get_balance(alice_wallet.address) > 0, "need ETH"
+```
+
+## 2. Alice publishes data asset
+
+In the same Python console:
+```python
+# Publish DATA datatoken, mint tokens
+from ocean_lib.web3_internal.currency import to_wei
+
+DATA_datatoken = ocean.create_data_token('DATA1', 'DATA1', alice_wallet, blob=ocean.config.metadata_cache_uri)
+DATA_datatoken.mint(alice_wallet.address, to_wei(100), alice_wallet)
+print(f"DATA_datatoken.address = '{DATA_datatoken.address}'")
+
+# Specify metadata & service attributes for Branin test dataset.
+# It's specified using _local_ DDO metadata format; Aquarius will convert it to remote
+# by removing `url` and adding `encryptedFiles` field.
+DATA_metadata = {
+    "main": {
+        "type": "dataset",
+        "files": [
+	  {
+	    "url": "https://raw.githubusercontent.com/trentmc/branin/main/branin.arff",
+	    "index": 0,
+	    "contentType": "text/text"
+	  }
+	],
+	"name": "branin", "author": "Trent", "license": "CC0",
+	"dateCreated": "2019-12-28T10:55:11Z"
+    }
+}
+DATA_service_attributes = {
+    "main": {
+        "name": "DATA_dataAssetAccessServiceAgreement",
+        "creator": alice_wallet.address,
+        "timeout": 3600 * 24,
+        "datePublished": "2019-12-28T10:55:11Z",
+        "cost": 1.0, # <don't change, this is obsolete>
+        }
+    }
+
+# Set up a service provider. We'll use this same provider for ALG
+from ocean_lib.data_provider.data_service_provider import DataServiceProvider
+provider_url = DataServiceProvider.get_url(ocean.config)
+# returns "http://localhost:8030"
+
+# Calc DATA service compute descriptor
+from ocean_lib.common.agreements.service_factory import ServiceDescriptor
+DATA_compute_service_descriptor = ServiceDescriptor.compute_service_descriptor(DATA_service_attributes, provider_url)
+
+#Publish metadata and service info on-chain
+DATA_ddo = ocean.assets.create(
+  metadata=DATA_metadata, # {"main" : {"type" : "dataset", ..}, ..}
+  publisher_wallet=alice_wallet,
+  service_descriptors=[DATA_compute_service_descriptor], # [("compute", {"attributes": ..})]
+  data_token_address=DATA_datatoken.address)
+print(f"DATA did = '{DATA_ddo.did}'")
+```
+
+Full details: [DATA_ddo](DATA_ddo.md)
+
+
+## 3. Alice publishes algorithm
+
+For this step, there are some prerequisites needed. If you want to replace the sample algorithm with an algorithm of your choosing, you will need to do some dependency management.
+You can use one of the standard [OCEAN algo_dockers images](https://github.com/oceanprotocol/algo_dockers) or publish a custom docker image.
+Use the image name and tag in the `container` part of the algorithm metadata.
+This docker image needs to have basic support for dependency installation e.g. in the case of Python, OS-level library installations, pip installations etc.
+Take a look at the [OCEAN tutorials](https://docs.oceanprotocol.com/tutorials/compute-to-data-algorithms/) to learn more about docker image publishing.
```

Comment:
"OCEAN tutorials" => "Ocean tutorials". Only use "OCEAN" in reference to the token.